src/Symfony/Component/Notifier/Bridge/Lox24/README.md

```diff
@@ -0,0 +1,73 @@
+LOX24 SMS Notifier
+==================
+
+Provides [LOX24 SMS Gateway](https://doc.lox24.eu/#tag/sms/operation/api_sms_post_collection) integration for Symfony
+Notifier.
+
+DSN example
+-----------
+
+```
+LOX24_DSN=lox24://USER:TOKEN@default?from=FROM&type=SERVICE_CODE&voice_language=VOICE_LANGUAGE&is_text_delete=DELETE_TEXT&callback_data=CALLBACK_DATA
```

Comment:
Looks like this comment was lost somehow but please change:
```suggestion
LOX24_DSN=lox24://USER:TOKEN@default?from=FROM&type=SERVICE_CODE&voice_language=VOICE_LANGUAGE&delete_text=DELETE_TEXT&callback_data=CALLBACK_DATA
```

Reply:
fixed

Reply:
delete_text please, it reads more natural